SQL_assignment/sql_assignment1

```diff
@@ -0,0 +1,82 @@
+CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+
+    Create table emp(EmpoyeeID int primary key,firstname varchar(20),lastname varchar(20),department varchar(20),salery int)
```

Comment:
why did you chose the first name and last name length as 20 ?

Reply:
I just initialized it with varchar so that only it only uses the required length and rest will be unused